plugins/generator-1.18.1/forge-1.18.1/procedures/block_schedule_tick.java.ftl

```diff
@@ -0,0 +1,2 @@
+world.getBlockTicks().schedule(new ScheduledTick<>(world.getBlockState(new BlockPos((int)${input$x}, (int)${input$y},(int)${input$z}))

```

Comment:
I think you should use LevelAccessor#scheduleTick as in 1.17.x here, it is more straightfoward